hibernate-core/src/test/java/org/hibernate/test/bytecode/enhancement/lazy/proxy/LazyGroupWithInheritanceTest.java

```diff
@@ -38,11 +42,25 @@
 @RunWith(BytecodeEnhancerRunner.class)
 @EnhancementOptions( lazyLoading = true )
 public class LazyGroupWithInheritanceTest extends BaseNonConfigCoreFunctionalTestCase {
+	public void loadEntityWithAssociationToAbstract() {
+		final Statistics stats = sessionFactory().getStatistics();
+		stats.clear();
+
+		inTransaction(
+				(session) -> {
+					final Order loaded = session.byId( Order.class ).load( 1 );
+					assert Hibernate.isPropertyInitialized( loaded, "customer" );
+					assertThat( stats.getPrepareStatementCount(), is( 1L ) );
+					assertThat( loaded, instanceOf( PersistentAttributeInterceptable.class ) );
+					final PersistentAttributeInterceptor interceptor = ((PersistentAttributeInterceptable) loaded).$$_hibernate_getInterceptor();
+					assertThat( interceptor, instanceOf( BytecodeLazyAttributeInterceptor.class ) );
+					final BytecodeLazyAttributeInterceptor interceptor1 = (BytecodeLazyAttributeInterceptor) interceptor;
```

Comment:
The method does not have a `@Test` annotation and the interceptor is unused. Did you forget to push some changes or was this added by accident?

Reply:
It was a gap in the original test.  I was only running it in IJ apparently (which does not care if the method is annotated with `@Test`)

Reply:
For the unused variable... meh